### ISSUE TITLE
Usage of the pouchdb-find plugin.

### ISSUE THREAD

**Issue by SwenVogel** [pouchdb-community/relational-pouch#92]
Hi, 

it is possible to use the pouchdb-find plugin in combination with relational-pouch?
I would like to use the features of the relational-pouch in combination with the
mongo like query syntax.

**Comment by broerse** [comment]
Yes we made it a Peer dependency in the last Beta. Also take a look at https://github.com/pouchdb-community/relational-pouch#dont-save-hasmany. We use find like this in [ember-pouch](https://github.com/pouchdb-community/ember-pouch).

**Comment by DrJonnie** [comment]
Thank's for your answer. Did this mean that the db.rel.find(type, options) method accepts
a mango like query as "option" paramater like described here: 
https://pouchdb.com/guides/mango-queries.html#query-language

**Comment by broerse** [comment]
Sorry but we only added it for [db.rel.findHasMany](https://github.com/pouchdb-community/relational-pouch#dbrelfindhasmanytype-belongstokey-belongstoid) because you need to create the `db.createIndex` and we have no plan for the normal find yet..

**Comment by yduartep** [comment]
So if I need to make queries over fields I can't use this relational-pouchdb library? I should use the standard pouchdb version with pouchdb-find?

**Comment by 509dave16** [comment]
@yduartep You have the right of it I think. `relational-pouch` does not offer any querying beyond providing document IDs and allDocs options to the different `find` methods. The `parseRelDocs` method of `relational-pouch` should be able to help you parse documents that you retrieve from `pouchdb-find`: https://github.com/pouchdb-community/relational-pouch#dbrelparsereldocstype-pouchdocs
